hack/update-code-gen.sh

```diff
@@ -28,7 +28,7 @@ export GOPATH=$(go env GOPATH)
 export PATH="${GOPATH}/bin:${PATH}"
 
 
-go install k8s.io/code-generator/cmd/deepcopy-gen
-go install github.com/maxbrunsfeld/counterfeiter
+go get k8s.io/code-generator/cmd/deepcopy-gen
+go get github.com/maxbrunsfeld/counterfeiter/v6
```

Comment:
What's this for?

Reply:
Partially just reverts https://github.com/google/go-containerregistry/pull/548/files#diff-87ee15ebb32e08fc4ef22a736ce9000d but our switch over to go modules basically broke assumptions in this file (Gopkg.toml does not vendor these packages anymore), and go modules require appending that v6 to the end of counterfeiter for reasons outside of my ability to comprehend. This + `go mod vendor` should fix the regression (I _think_).